automation/impossible-to-read-carbonited-metadata-directly-from-within-workflow-check.sh

```diff
@@ -43,18 +43,18 @@ launchSubmission dumbledore.admin@test.firecloud.org cromwell-tests-billing-proj
 findLastSubmissionID dumbledore.admin@test.firecloud.org cromwell-tests-billing-project cromwell-tests-workspace
 submission4=$submissionID
 
-waitForSubmissionAndWorkflowStatus 20 Done Failed dumbledore.admin@test.firecloud.org cromwell-tests-billing-project cromwell-tests-workspace "$submission1"
+waitForSubmissionAndWorkflowStatus 120 Done Failed dumbledore.admin@test.firecloud.org cromwell-tests-billing-project cromwell-tests-workspace "$submission1"
 findFirstWorkflowIdInSubmission dumbledore.admin@test.firecloud.org cromwell-tests-billing-project cromwell-tests-workspace "$submission1"
 checkIfWorkflowErrorMessageContainsSubstring dumbledore.admin@test.firecloud.org cromwell-tests-billing-project cromwell-tests-workspace "$workflowID" "$expectedErrorText"
 
-waitForSubmissionAndWorkflowStatus 20 Done Failed dumbledore.admin@test.firecloud.org cromwell-tests-billing-project cromwell-tests-workspace $submission2
+waitForSubmissionAndWorkflowStatus 2 Done Failed dumbledore.admin@test.firecloud.org cromwell-tests-billing-project cromwell-tests-workspace $submission2
```

Comment:
Curious: Why does the earlier diff change from `20` to `120` and this changes from `20` to `2`?

Reply:
All the submissions are being submitted roughly at the same time, and then we start waiting for them to finish. So I thought if we've already been waiting for the first one for 2 hours and it didn't succeed, it wouldn't make much sense to wait for other submissions any longer - so only 2 attempts for them (the second one just in case to account for random network glitches or something).

Reply:
seems like a great opportunity to add the double quotes that `submission2` is missing relative to the other invocations above and below 😄

Reply:
We could still get unlucky and have this first one finish quickly while the others take a bit longer. In other words, we might not have already waited for 2 hours by the time we get here.